.travis.yml

```diff
@@ -8,31 +7,20 @@ cache:
   directories:
     - ~/.cache/pip
 env:
-  - TARGET: bashate
-  - TARGET: pep8
-  - TARGET: docs
-  - TARGET: docs-gnocchi.xyz
-
-  - TARGET: py27-mysql-ceph-upgrade-from-3.1
-  - TARGET: py35-postgresql-file-upgrade-from-3.1
-
-  - TARGET: py27-mysql
-  - TARGET: py35-mysql
-  - TARGET: py27-postgresql
-  - TARGET: py35-postgresql
+  - TARGETS: pep8,docs,docs-gnocchi.xyz,py27-mysql-ceph-upgrade-from-3.1,py35-postgresql-file-upgrade-from-3.1
```

Comment:
hmm... yeah, those pyxx builds are going to be massive... and sequential? our are jobs right now only one by one as well?

i'd prefer if we split the pyxx jobs but if it's faster than i can live with massive logs.